docs/Concepts/Permissioning/Onchain-Permissioning.md

```diff
@@ -56,6 +56,15 @@ Permissioning implements three allowlists:
     If using account permissioning and privacy, a signing key must be specified using the
     [`--privacy-marker-transaction-signing-key-file`] command line option and the signing key
     included in the accounts allowlist.
+    
+!!! tip 
+  
+    If nodes are not connecting as expected, set the [log level to `TRACE`](../../Reference/CLI/CLI-Syntax.md#logging)
+    to identify the issue. 
```

Comment:
Probably worth adding a bit more info on how to find problems. Something like:


> If nodes are not connecting as expected, set the [log level to `TRACE`](../../Reference/CLI/CLI-Syntax.md#logging) and search for messages containing `Node permissioning` to identify the issue.

Reply:
Updated